package.json

```diff
@@ -23,6 +23,7 @@
     "change": "beachball change --scope \"!packages/fluentui/*\"",
     "check-for-modified-files": "cd scripts && just-scripts check-for-modified-files",
     "checkchange": "beachball check  --scope \"!packages/fluentui/*\" --changehint \"Run 'yarn change' to generate a change file\"",
+    "check-if-packages-affected": "node ./scripts/monorepo/checkIfPackagesAffected.js",
```

Comment:
can we rename this to express proper intent of this script ?

something like: `check-if-one-of-component-libraries-is-affected` or `check-if-v0-v8-v9-is-affected` ... naming is hard right :D